fast64_internal/f3d/f3d_material.py

```diff
@@ -2460,45 +2460,54 @@ class RDPSettings(bpy.types.PropertyGroup):
         name="Z Buffer",
         default=True,
         update=update_node_values_with_preset,
+        description="Turns on/off Z-Buffer.  Z-Buffer set to 0 if disabled."
     )
     g_shade: bpy.props.BoolProperty(
         name="Shading",
         default=True,
         update=update_node_values_with_preset,
+        description="Turns on/off shading. Shade register set to 0 if disabled."
     )
     # v1/2 difference
     g_cull_front: bpy.props.BoolProperty(
         name="Cull Front",
         update=update_node_values_with_preset,
+        description="Turns on/off drawing of front faces"
     )
     # v1/2 difference
     g_cull_back: bpy.props.BoolProperty(
         name="Cull Back",
         default=True,
         update=update_node_values_with_preset,
+        description="Turns on/off drawing of back faces"
     )
     g_fog: bpy.props.BoolProperty(
         name="Fog",
         update=update_node_values_with_preset,
+        description="Turns on/off fog calculation. Fog variable gets stored into shade alpha"
     )
     g_lighting: bpy.props.BoolProperty(
         name="Lighting",
         default=True,
         update=update_node_values_with_preset,
+        description="Enables calculation of shade values from lights and vertex normals. Turn off for vertex colors"
     )
     g_tex_gen: bpy.props.BoolProperty(
         name="Texture UV Generate",
         update=update_node_values_with_preset,
+        description="Generates texture coordinates that maps from norm x/y to [0-1]x/y"
```

Comment:
This is technically accurate but isn't going to give any information to someone who doesn't already know what this does. I'd suggest something like "Generates texture coordinates for reflective objects, based on vertex normals and lookat directions. In this version, the sky is at the center of your texture image and the ground is a circle inscribed in the border."